vendor/github.com/kubeedge/beehive/pkg/common/util/parse_resource.go

```diff
@@ -13,6 +13,8 @@ func ParseResourceEdge(resource string, operation string) (string, string, strin
 	resourceSplits := strings.Split(resource, "/")
 	if len(resourceSplits) == 3 {
 		return resourceSplits[0], resourceSplits[1], resourceSplits[2], nil
+	} else if len(resourceSplits) == 5 {
```

Comment:
We need to change in https://github.com/kubeedge/beehive and then update the vendor.